### ISSUE TITLE
 Lighthouse run unable to kill child processes (similar problem as issue 6306)

### ISSUE THREAD

**Issue by dzmitrykashlach** [GoogleChrome/lighthouse#13871]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

I am facing this issue on internal system. What should I put here ?

### What happened?

This issue is created in order to display, that [issue 6306](https://github.com/GoogleChrome/lighthouse/issues/6306) still exists.
I am facing this issue on our internal system, so I do not know, what to put in "URL" field
Additionally, I was running test with custom gatherer & audit.
`yarn run lighthouse --config-path=custom-config.js https://wiki-stag.wdf.sap.corp/wiki/dashboard.action#recently-worked --view
yarn run v1.22.18`

### What did you expect?

Lighthouse finishes task without exception.

### What have you tried?

AFAIK, it's a known issue
https://github.com/GoogleChrome/lighthouse/issues/6306

### How were you running Lighthouse?

CLI

### Lighthouse Version

6.0.0

### Chrome Version

100.0.4896.88

### OS

Windows

### Relevant log output

```shell
Lighthouse is not finishing test correctly and throws error:
`$ yarn run lighthouse --config-path=custom-config.js https://wiki-stag.wdf.sap.corp/wiki/dashboard.action#recently-worked --view
yarn run v1.22.18
warning package.json: No license field
$ C:\src\nft-utils\googlelighthouse\node_modules\.bin\lighthouse --config-path=custom-config.js https://wiki-stag.wdf.sap.corp/wiki/dashboard.action#recently-worked --view
Tue, 19 Apr 2022 14:43:25 GMT LH:ChromeLauncher Waiting for browser.
Tue, 19 Apr 2022 14:43:25 GMT LH:ChromeLauncher Waiting for browser...
Tue, 19 Apr 2022 14:43:26 GMT LH:ChromeLauncher Waiting for browser.....
Tue, 19 Apr 2022 14:43:26 GMT LH:ChromeLauncher Waiting for browser.....√
Tue, 19 Apr 2022 14:43:26 GMT LH:config:warn unrecognized audit in 'onlyAudits': universalheader-audit
Tue, 19 Apr 2022 14:43:26 GMT LH:status Connecting to browser
Tue, 19 Apr 2022 14:43:26 GMT LH:status Resetting state with about:blank
Tue, 19 Apr 2022 14:43:27 GMT LH:status Benchmarking machine
Tue, 19 Apr 2022 14:43:27 GMT LH:status Initializing…
Tue, 19 Apr 2022 14:43:27 GMT LH:status Running defaultPass pass UniversalHeaderGatherer
Tue, 19 Apr 2022 14:43:27 GMT LH:status Resetting state with about:blank
Tue, 19 Apr 2022 14:43:27 GMT LH:status Setting up network for the pass trace
Tue, 19 Apr 2022 14:43:27 GMT LH:status Beginning devtoolsLog and trace
Tue, 19 Apr 2022 14:43:27 GMT LH:status Loading page & waiting for onload
Tue, 19 Apr 2022 14:43:38 GMT LH:status Gathering in-page: UniversalHeaderGatherer
Tue, 19 Apr 2022 14:43:38 GMT LH:status Gathering devtoolsLog & network records
Tue, 19 Apr 2022 14:43:38 GMT LH:status Gathering: UniversalHeaderGatherer
Tue, 19 Apr 2022 14:43:38 GMT LH:status Disconnecting from browser...
Tue, 19 Apr 2022 14:43:38 GMT LH:status Analyzing and running audits...
Tue, 19 Apr 2022 14:43:39 GMT LH:status Auditing: UniversalHeaderAudit
Tue, 19 Apr 2022 14:43:39 GMT LH:status Generating results...
Tue, 19 Apr 2022 14:43:39 GMT LH:Printer html output written to C:\src\nft-utils\googlelighthouse\wiki-stag.wdf.sap.corp_2022-04-19_17-43-27.report.html
Tue, 19 Apr 2022 14:43:39 GMT LH:ChromeLauncher Killing Chrome instance 4900
Tue, 19 Apr 2022 14:43:40 GMT LH:ChromeLauncher:warn Chrome could not be killed Command failed: taskkill /pid 4900 /T /F
ERROR: The process with PID 10376 (child process of PID 4900) could not be terminated.
Reason: There is no running instance of the task.

Tue, 19 Apr 2022 14:43:40 GMT LH:ChromeLauncher Killing Chrome instance 4900
Tue, 19 Apr 2022 14:43:40 GMT LH:ChromeLauncher:warn Chrome could not be killed Command failed: taskkill /pid 4900 /T /F
ERROR: The process with PID 18768 (child process of PID 4900) could not be terminated.
Reason: There is no running instance of the task.
ERROR: The process with PID 17224 (child process of PID 4900) could not be terminated.
Reason: There is no running instance of the task.
ERROR: The process with PID 4900 (child process of PID 12572) could not be terminated.
Reason: There is no running instance of the task.

Runtime error encountered: Couldn't quit Chrome process. Error: Chrome could not be killed Command failed: taskkill /pid 4900 /T /F
ERROR: The process with PID 10376 (child process of PID 4900) could not be terminated.
Reason: There is no running instance of the task.

Error: Couldn't quit Chrome process. Error: Chrome could not be killed Command failed: taskkill /pid 4900 /T /F
ERROR: The process with PID 10376 (child process of PID 4900) could not be terminated.
Reason: There is no running instance of the task.

    at C:\src\nft-utils\googlelighthouse\node_modules\lighthouse\lighthouse-cli\run.js:178:11
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runLighthouse (C:\src\nft-utils\googlelighthouse\node_modules\lighthouse\lighthouse-cli\run.js:216:5)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by brendankenny** [comment]
You mention lighthouse-batch in the issue title (and that's what the referenced issue #6306 was about) but not in the quoted logs. If this is `lighthouse-batch`, you should file an issue there: https://github.com/mikestead/lighthouse-batch

If it's just plain Lighthouse that's failing:
- can you try an updated version of Lighthouse? 6.0.0 is from two years ago. I can't think of any particular changes that would fix this, but it'll be easier to diagnose with an up to date version
- can you tell if Chrome has actually been killed? That would tell us if Lighthouse is trying to kill a Chrome that's already been killed, trying to kill something it's not allowed to, and/or trying to kill the wrong process.
- is the report output being successfully written to disk? If so, a workaround would be to ignore the exit code for now while we figure out the problem

**Comment by dzmitrykashlach** [comment]
1) yes, I've upgraded to 9.5.0. FYI, I'm using **yarn** for running & managin lighthouse version(hope, this info is helpful).

2) Actually Chrome instance is killed, but exception is still in console ouput;
3) yes, report is written to disk.

One small question additionally: with 6.0.0 I was using `emulatedFormFactor:'desktop'` setting in `custom-config.js`. With 9.5.0 the same setting is not working and LH is starting in mobile mode. How to override this behavior back to desktop mode ?

**Comment by paulirish** [comment]
I think this has been resolved with #14070. this hasn't shipped in a release yet tho.
